core/src/main/java/org/infinispan/manager/CacheContainer.java

```diff
@@ -19,4 +20,13 @@
    @Override
    <K, V> Cache<K, V> getCache(String cacheName);
 
+   /**
+    * Provides access to administrative methods which affect the underlying cache container, such as cache creation and
+    * removal. If the underlying container is clustered or remote, the operations will affect
```

Comment:
Think you missed finishing this sentence

Reply:
ok